### DISCUSSION TITLE
URLs in github action logs that are followed by a comma should be parsed as links that do not include the comma

### DISCUSSION THREAD

**Discussion by hugobuddel** [community/community#109477] [Actions]
### Select Topic Area

Bug

### Body

Any URL in a github action log that is followed by a comma is parsed as if the comma belongs to the URL. Usually the comma should not be part of the URL. It is therefore not possible to directly follow the hyperlink; it is necessary to copy the link and remove the comma first.

In other places (like issues), the comma is not parsed as part of the URL. So it makes sense to also not include the comma in github action logs.

**Comment by thisisthetechie** [comment]
Azure DevOps has a similar issue where a trailing quote mark is always added to the link...
